filebeat/module/apache2/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- key: apache2
+  title: "apache2"
```

Comment:
This is created by the generator, which is somehow limited in that regard. Do you mean fixing it up by hand?